File: LP2/madebyprofessor/Codigos_Aulas_02_03_Java/Arquivos Cpp/Media.cpp
#include <iostream>

int main() {
    double n1 = 0.0;
    double n2 = 0.0;

    std::cout << "Primeira nota: ";
    std::cin >> n1;

    std::cout << "Segunda nota: ";
    std::cin >> n2;

    double media = (n1 + n2) / 2.0;

    if (media >= 7.0) {
        std::cout << "Média: " << media << " — Aprovado\n";
    } else {
        std::cout << "Média: " << media << " — Em recuperação\n";
    }

    return 0;
}

File: LP2/Media.java
import java.util.Scanner;


public class Media{

    public static void main(String[] args){
         
        double media;

        double n1;
        double n2;

        try(Scanner scan = new Scanner(System.in)){
            System.out.print("Primeira nota: ");
            n1 = scan.nextDouble();
            System.out.print("Segunda nota: ");
            n2 = scan.nextDouble();
        
            media = (n1 + n2) / 2.0;
            String situacao = (media >= 7.0) ? "Aprovado" : "Em recuperação";
            
            System.out.printf("Média: %.2f - %s%n", media, situacao);

        }

    }

}